src/basic/generate-bpf-delegate-configs.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# Convert the bpf_{cmd,map_type,prog_type,attach_type} enums into a string
+# array to be used as configuration options
+
+import re
+import sys
+
+def print_usage_and_exit():
+    print(f'Usage: {sys.argv[0]} header <header>')
```

Comment:
I thought this generates a self-contained header file, but it does not, and generates .inc file.
https://github.com/systemd/systemd/pull/36134/commits/98f2e99850eb01d8e7bb8f3c3f4dc79fb2f8e986#r2169349245
Then, I think `code` is better. Sorry.